chains/metadata.yaml

```diff
@@ -2166,6 +2191,31 @@ osmosis:
   slip44: 118
   transactionOverrides:
     gasPrice: "0.025"
+piccadilly:
```

Comment:
intentional? I don't see it in the `chains` dir

Reply:
yup it's here https://github.com/hyperlane-xyz/hyperlane-registry/tree/pb/oort/chains/piccadilly

Reply:
from https://github.com/hyperlane-xyz/hyperlane-registry/pull/156